build.make

```diff
@@ -72,6 +72,11 @@ endif
 # toolchain.
 BUILD_PLATFORMS =
 
+# PULL_BASE_REF will contain the base ref that was pushed to - for instance,
+# master or release-0.2 for a PR merge, or v0.2 for a tag.
+# See: https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/README.md
+PULL_BASE_REF =
```

Comment:
"will contain the base ref" is a bit misleading. It only is non-empty if set explicitly when invoking make with `make PULL_BASE_REF=...`, which is not always the case.

`PULL_BASE_REF=... make` leads to an empty `PULL_BASE_REF` make variable. It doesn't matter at the moment because only `push-multiarch` depends on the variable but I find that a bit dangerous because it might bite us in the future when that changes.

Reply:
I took the description of `PULL_BASE_REF` from https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/README.md#custom-substitutions as the comment for this variable, I understand now why it might be dangerous